media/js/mozorg/contact-spaces.js

```diff
@@ -311,6 +322,8 @@
                 $('#nav-spaces, #meta-spaces').fadeOut(100, function () {
                     $('#nav-communities, #meta-communities').show();
                 });
+            } else if (tab === 'contact') {
+                $('#nav-spaces, #meta-spaces, #nav-communities, #meta-communities').fadeOut(100);
```

Comment:
For consistency, let's keep the logic uniform and make the condition here `(tab === 'contact' && wideMode)`. Then we can simplify/ the `toggleMobileNav` function below to be:

```
} else if (tab === 'contact' && !wideMode) {
    $('#nav-communities-select, #meta-communities, #nav-spaces-select, #meta-spaces').hide();
}
```